centrallix/tests/test_moneytype_00.c

```diff
@@ -0,0 +1,30 @@
+#include "obj.h"
+#include <assert.h>
+
+long long
+test(char** name)
+{
+    *name = "moneytype_00 - obj_internal_FormatMoney";
```

Comment:
Since this is technically testing objFormatMoneyTmp, not obj_internal_FormatMoney, could you change the name to reflect that?

Reply:
The changes were to obj_internal_FormatMoney, and objFormatMoneyTmp is basically a wrapper that allocates a buffer and calls obj_internal_FormatMoney.  The point of the test is to see if ob_internal_FormatMoney is working properly, even though it calls objFormatMoneyTmp.  I'll add a comment explaining that.

Reply:
One unit testing best practice I forgot to mention to you is that you generally want to test the public interface - who cares how it's implemented under the hood so long as the functions that will be called elsewhere give you the right results. (If you can change your implementation without breaking your clients, your unit tests also shouldn't break!) So in this case, it doesn't really matter that objFormatMoneyTmp works under the hood by calling obj_internal_FormatMoney, or that you had to edit that function specifically. All an external caller cares is that objFormatMoneyTmp spits out what it's supposed to, so that should be the focus of testing. 

So you wrote these tests right by calling objFormatMoneyTmp, but I think it would be best for the name to also reflect that. Does that make sense?